Compilation broken on Windows with `utp_shutdown` changes

```
utp_internal.cpp: In function 'void utp_shutdown(UTPSocket*, int)':
utp_internal.cpp:3410:13: error: 'SHUT_WR' was not declared in this scope
  if (how != SHUT_WR) {
             ^~~~~~~
utp_internal.cpp:3413:13: error: 'SHUT_RD' was not declared in this scope
  if (how != SHUT_RD) {
             ^~~~~~~
```

--- a/utp_types.h
+++ b/utp_types.h
@@ -49,6 +49,9 @@
 	#include <winsock2.h>
 	#include <ws2tcpip.h>
 	#define IP_OPT_DONTFRAG IP_DONTFRAGMENT
+	#define SHUT_RD SD_RECEIVE
+	#define SHUT_WR SD_SEND
+	#define SHUT_RDWR SD_BOTH
 #else
 	#include <netinet/in.h>
 	#include <arpa/inet.h>

